### PR TITLE
Added a note title

### PR DESCRIPTION
**Pull request by namolnad**
@mhanlon I've added a title (as well as a creation date) into the dateModel and have introduced it into the note detailView. Let me know if you think it's ready to merge into master. Thanks!

### CODE REVIEW

File: blanknote/blanknote/DetailViewController.m [hunk 1]
```diff
@@ -35,35 +35,40 @@ - (void)configureView
...
     }
-    
-    NSString *firstFiveLettersOfNote = @"";
-    if (self.noteTextView.text.length > 4)
-    {
-        firstFiveLettersOfNote = [[self.noteTextView.text substringToIndex:5] capitalizedString];
-        self.title = [NSString stringWithFormat:@"%@... (modified %@)", firstFiveLettersOfNote, timeStamp];
-    }
-    else if (self.noteTextView.text.length > 0)
+    if (noteTitle && ![noteTitle isEqualToString:@""])
```
Comment (mhanlon) [inline blanknote/blanknote/DetailViewController.m]: Good, another common way to test for a **not** empty string is to do this:
```
if ( noteTitle && noteTitle.length > 0 )
```
(It's marginally faster to compare the length of the string to 0 than it is to compare two strings... but, like I said, marginally.)

File: blanknote/blanknote/DetailViewController.m [hunk 2]
```diff
@@ -72,6 +77,9 @@ - (void)viewDidLoad
     [super viewDidLoad];
     // Do any additional setup after loading the view, typically from a nib.
     [self configureView];
+    [self setNavTitle];
+    self.textViewText = [[NSString alloc] init];
+    self.textFieldText = [[NSString alloc] init];
```
Comment (mhanlon) [inline blanknote/blanknote/DetailViewController.m]: You could probably just leave these as `nil`, to start, no need to allocate a new empty string right off the bat.
